### PR TITLE
fix(gate): take into position left when calculating liquidation size

### PR DESCRIPTION
**Pull request by pcriadoperez**
Note: Was not able to test

### CODE REVIEW

**Comment by carlosmiei** [comment]
@pcriadoperez any update here?

**Comment by pcriadoperez** [comment]
gate.fetchLiquidations (BTC/USDT:USDT)
2023-11-17T00:25:10.425Z iteration 0 passed in 443 ms

       symbol | contracts | contractSize | price | baseValue | quoteValue |     timestamp |                 datetime
--------------------------------------------------------------------------------------------------------------------
BTC/USDT:USDT |        99 |              | 36205 |    0.0099 |   358.4295 | 1700178349000 | 2023-11-16T23:45:49.000Z
1 objects

File: ts/src/test/static/request/gate.json [hunk 1]
```diff
@@ -215,6 +215,17 @@
...
+                "method": "fetchLiquidations",
+                "url": "https://api.gateio.ws/api/v4/futures/usdt/liq_orders?contract=BTC_USDT",
+                "input": [
+                  "BTC/USDT:USDT"
+                ],
+                "output": "[{info:{contract:\"BTC_USDT\",left:\"0\",size:\"-99\",fill_price:\"36205\",order_price:\"36348.5\",time:\"1700178349\",},symbol:\"BTC/USDT:USDT\",contracts:99,contractSize:undefined,price:36205,baseValue:0.0099,quoteValue:358.4295,timestamp:1700178349000,datetime:\"2023-11-16T23:45:49.000Z\",},{info:{contract:\"BTC_USDT\",left:\"0\",size:\"-23\",fill_price:\"36295.2\",order_price:\"36470.3\",time:\"1700181591\",},symbol:\"BTC/USDT:USDT\",contracts:23,contractSize:undefined,price:36295.2,baseValue:0.0023,quoteValue:83.47896,timestamp:1700181591000,datetime:\"2023-11-17T00:39:51.000Z\",},{info:{contract:\"BTC_USDT\",left:\"0\",size:\"-109\",fill_price:\"36295.2\",order_price:\"36422.3\",time:\"1700181591\",},symbol:\"BTC/USDT:USDT\",contracts:109,contractSize:undefined,price:36295.2,baseValue:0.0109,quoteValue:395.61768,timestamp:1700181591000,datetime:\"2023-11-17T00:39:51.000Z\",},]"
```
Comment (carlosmiei) [inline ts/src/test/static/request/gate.json:227]: @pcriadoperez this is not right 😅  I think you mixed request+response tests no?

Comment (pcriadoperez) [reply ts/src/test/static/request/gate.json]: sorry I think here was figuring out the tests, now it should be fixed and added the response one too

**Pull request merged**